docs/breaking_changes.md

```diff
@@ -0,0 +1,35 @@
+<!--[metadata]>
++++
+aliases = ["/engine/misc/breaking/"]
+title = "Engine breaking changes and incompatibilities"
+description = "Breaking changes and incompatibilities"
+keywords = ["docker, documentation, about, technology, breaking", "incompatibilities"]
+[menu.main]
+parent = "engine_use"
+weight=80
++++
+<![end-metadata]-->
+
+# Breaking changes and incompatibilities
+
+Every Engine release strives to be backward compatible with its predecessors. In
+all cases, feature removal will be communicated two releases in advance and
+documented as part of the [deprecated features](deprecated.md) page.
+
+Nevertheless, Docker is a fast moving project, and newly introduced features may
+sometime introduce incompatibilities. This page documents such incompatibilities
+for each Engine version.
+
+# Engine 1.10.0
+
+## Distribution
+
+- Images pushed by Engine 1.10.0 to a Registry 2.3 cannot be pulled by digest by
+  previous Engine versions. Because Docker Content Trust heavily relies on this
+  feature, this in turns prevents images pushed by Engine 1.10.0 to be pulled by
+  older Engine versions with Docker Content Trust enabled.
```

Comment:
Maybe that should read `this in turns prevents images pushed by Engine 1.10.0 to be pulled by
 older Engine versions with Docker Content Trust enabled against a private Registry version 2.3 or higher`.